File: WordApp/Models/TaskModels/SentenceTaskModels/OrderedSentenceTaskModel.cs
using WordApp.Models.Base;
using WordApp.Models.Sentences;

namespace WordApp.Models.TaskModels.SentenceTaskModels
{
    public class OrderedSentenceTaskModel: BaseModel
    {
        public int Order { get; set; }
        public SentenceModel Sentence { get; set; }
    }
}

File: WordApp/ClientApp/src/app/models/tasks/sentences/ordered-sentence-task.model.ts
import { SentenceModel } from './../../sentences/sentence.model';

export class OrderedSentenceTaskModel {
  order: number;
  sentence: SentenceModel;
  id: string;
  rowVersion: string;

  constructor(order: number, sentence: SentenceModel, id: string, rowVersion: string) {
    this.id = id;
    this.rowVersion = rowVersion;
    this.order = order;
    this.sentence = sentence;
  }
}


